bindings/python/tests/test_block.py

```diff
@@ -214,24 +216,68 @@ def test_block():
                                                          "signature": "0x30cb012af3402be1b4b2ed18e2aba86839da06ba38ff3277c481e17c003f0199ba26f5613199e0d24035628bb2b69a6ea2a7682e41c30244996baf3a2adc1c00"}},
                                           {"type": 1,
                                            "reference": 0}]}}
-    block = Block.from_dict(block_dict)
+    block = BlockWrapper.from_dict(block_dict)
```

Comment:
The above data does not contain the header?

Reply:
sorry ... that's just testing a `BasicBlock`.